src/ui-lib/sass/05-components/03-organisms/_list-image-links.scss

```diff
@@ -1,8 +1,9 @@
 .grav-c-list-image-links {
+  $image-width-m: calc(100% * (1/6) - #{$grav-sp-m});
+  $image-width-l: calc(100% * (1/6) - #{$grav-sp-l});
 
-  padding: $grav-sp-l $grav-sp-m calc(#{$grav-sp-l} - #{$grav-sp-m});
-  background-color: $grav-co-neutral-ashtray;
-  column-gap: $grav-sp-l;
+  padding: 0 $grav-sp-xl $grav-sp-l;
```

Comment:
If you address the padding of the container as explained above, you don't even need to set the padding on this element, and let the margin-top deal with the vertical-spacing (which is already set and you don't need to do anything). The idea being is that every element should be context-free, and the container should dictate how things are set. There are few exceptions to this, but this is the overall ideal.